manifests/i/Initex/YogaDNS/1.45/Initex.YogaDNS.installer.yaml

```
@@ -1,20 +1,19 @@
-# Created with komac v2.1.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
-
-PackageIdentifier: Initex.YogaDNS
-PackageVersion: '1.44'
-InstallerLocale: en-US
-InstallerType: inno
-Scope: machine
-InstallModes:
-- interactive
-UpgradeBehavior: install
-FileExtensions:
-- xml
-ReleaseDate: 2024-02-20
-Installers:
-- Architecture: x86
-  InstallerUrl: https://www.yogadns.com/download/YogaDNSSetup.exe
-  InstallerSha256: C1DDFC723F8688C9C2E43A4A66E8E31DB2A592007BECA26429EE93FAAD0CBFDA
-ManifestType: installer
-ManifestVersion: 1.6.0
+# Created with YamlCreate.ps1 v2.4.1 $debug=AUSU.CRLF.5-1-26100-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Initex.YogaDNS
+PackageVersion: "1.45"
+InstallerLocale: en-US
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+UpgradeBehavior: install
+FileExtensions:
+- xml
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```
Testing in an x64 win11 sandbox didn’t prompt the driver